### ISSUE TITLE
Add a report of recent activity

### ISSUE THREAD

**Issue by DavidRoy** [BiologicalRecordsCentre/ukotas#35]
A report page to summarise recent activity across whole site
- list of species of added for each territory
- change in establishment status

Abilty to email summary to those who register to receive

**Comment by DavidRoy** [comment]
@johnvanbreda could you give your view on the complexity of this and resource required

**Comment by johnvanbreda** [comment]
Providing a log view which shows all revisions of all species accounts, filterable by territory, can be done with a Drupal core view using content revisions. It would just be a big table of columns for each field showing the field value in that version, plus fields for the change date, sorted by change date. This view would not highlight the specific changed field, so it would contain rows for all changes, not just establishment status. We could include a column for the original creation date of the account so you can see if it is new. This is a few hours work. A link to a filtered version of this page showing the revisions just for one species account could be added to the species account page.

Making a version of this which only shows rows when the establishment status changes is quite complex and needs a custom solution, probably several days work.

We would also need to develop custom code for signing up to a daily summary of changes - presumably this would allow signup to changes just for a given territory or list of territories. This would also be 2-3 days work.

**Comment by DavidRoy** [comment]
Thanks John. So if we just went for the first requirement to summarise species added, the work involved would be:
1. a couple of hours to show a filtered list of new pages by date (i.e. species added)
2. 2-3 days for a custom module for a mechanism to sign upto summary of changes by territory

@johnvanbreda is that right?

The change in establishment status is a lower priority

**Comment by johnvanbreda** [comment]
That's correct. Point 1 may take only an hour or so if it's just a simple list of species accounts, the territory and the date added, rather than a log of changes.
